### ISSUE TITLE
Configuration changes on the fly

### ISSUE THREAD

**Issue by gallipi** [mvisonneau/gitlab-ci-pipelines-exporter#825]
This is a suggestion in case it is of interest.

Periodically we need to add new groups in the wildcard to be able to retrieve their metrics.
It would be great if the exporter could detect these changes and apply them without having to be restarted.

Thank you for all your effort and dedication.

**Comment by NoeFaucher** [comment]
Yeah maybe like Prometheus [that can reload its configuration at runtime](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration).
